cli/session.go

```diff
@@ -71,16 +80,15 @@
 	DefaultSessionFolder = fmt.Sprintf("%s/.clouditor/", home)
 }
 
-func NewSession(url string, opts ...grpc.DialOption) (session *Session, err error) {
+func NewSession(url string) (session *Session, err error) {
```

Comment:
since CLI is only used on localhost, you removed opts?

Reply:
I saw no use case for additional options. You specify the host in the url param. We did not use them, even in the unit tests.